dist/init/linux-upstart/caddy.conf

```diff
@@ -14,5 +15,7 @@ respawn limit 10 5
 limit nofile 4096 4096
```

Comment:
This is equivalent to `ulimit -n` right? We should raise this as high as it can go, FWIW.